app/models/concerns/active_storage_file.rb

```diff
@@ -9,7 +9,7 @@ module ActiveStorageFile
     has_one_attached :file
 
     def assign_attributes(new_attributes)
-      if SettingsHelper.feature_on?(:active_storage)
+      if SettingsHelper.active_storage_enabled?
```

Comment:
Do we need this feature flag here?  I can't recall if we need it for some reason or perhaps it was left over from a previous iteration of the code, before we separated things into modules

Reply:
I also can't recall. It passes CI with it removed though, so I suppose it's fine to leave it out. I recall this method causing a lot of problems, so likely extra stuff was left in it while working up to a solution.